### DISCUSSION TITLE
Add toggle for homekit friendly naming

### DISCUSSION THREAD

**Discussion by mouth4war** [jseidl/magic-areas#292] [Ideas / Feature Requests]
Accent lights (Bedroom) should be replaced with Bedroom Accent Lights when enabled.

**Comment by jseidl** [comment]
I could expose a name template, but how would this be different than renaming the entity?

**Reply by mouth4war** [reply]
Painful to do it for every room and light group. Maybe you can just change the naming?

**Comment by jseidl** [comment]
I'd rather keep the default naming as it's more an opinion / taste thing while allowing users to change it themselves.

I think I can add on the area config a name pattern that you can change and change all the entities, that should help with manually renaming all of them

**Comment by jseidl** [comment]
I'm circling back on this and since I started adding area devices (still on beta) it does make sense to change the naming to this as home assistant also makes displaying those more friendly by removing the device name, although it will be a breaking change.

**Comment by jseidl** [comment]
This is implemented in release 4.0.0 which is out! No toggle, it's just the default now!
